docs/sources/http_api/datasource_permissions.md

```diff
@@ -218,6 +261,16 @@ Status codes:
 
 Removes the permission with the given `permissionId` for the data source with the given `id`.
 
+### Required permissions
+
+See note in the [introduction]({{< ref "#data-source-permissions-api" >}}) for an explanation.
+
+| Action                         | Scope                                                                        |
+| ------------------------------ | ---------------------------------------------------------------------------- |
+| datasources.permissions:delete | datasources:\*<br>datasources:id:\*<br>datasources:id:1 (single data source) |
```

Comment:
I know this is the pattern we have used so far, however just thought that it might not be clear that there is an OR in this set of scopes. How can we make it more clear for a reader that you need one of those options?

Not suggesting to change anything in this PR, just opening up a discussion.